koregraph/managers/choregraphy.py

```diff
@@ -1,18 +1,11 @@
 from datetime import timedelta
-from pickle import load
+import pickle
```

Comment:
Prefer `from module import function` as it is lighter.

If `load` is not clear enough as a function name, `... import load as load_pickle`.